tests/api.spec.ts

```diff
@@ -2,17 +2,29 @@ import pick from 'lodash.pick';
 
 import { baseURL, sessionToken, source, TEST_TIMEOUT } from './constants/base';
 import { bundleFiles, bundleFilesFull, singleBundleFull } from './constants/sample';
-import { getFilters, createBundle, checkBundle, extendBundle, getAnalysis, AnalysisStatus } from '../src/http';
+import {
+  getFilters,
+  createBundle,
+  checkBundle,
+  extendBundle,
+  getAnalysis,
+  AnalysisStatus,
+  initReport,
+} from '../src/http';
 import { BundleFiles } from '../src/interfaces/files.interface';
 import * as needle from '../src/needle';
 import { gunzipSync } from 'zlib';
 
-const fakeBundleHash = 'd9f1171fb6f6bf12eb217fee43eef3cebd6a85cc78bc333035bf4cc3ebf1cf68';
+//TODO: these tests need to be refactored, they keep failing every time there's changes to deeproxy
```

Comment:
I don't think this has anything to do with deeproxy? What really changed here - are we now uploading more files because the filters have changed?

Reply:
This comment was just because we usually have to change the hashes and/or files for tests to work in `code-client` sometimes when there are changes in `deeproxy` (for example in [this PR](https://github.com/snyk/code-client/pull/167/files#diff-08d4fa826f385817cef2fe4fceb8ed9aebf88bf61f1f7c393b634bad04d1a4bdR10) by Matt).

It's nothing urgent tbh